views/styles.html

```diff
@@ -1,12 +1,16 @@
 {% extends "_donut.html" %}
 
-{% block pageTitle %}{{pageTitle}}{% endblock %}
-
 {% block pageHead %}
   {{ super() }}
 {% endblock %}
 
 {% block main %}
+  <section class="_style-guide--group">
```

Comment:
Made some changes to try and match what we had on `wox-ui` but we will still need to create stories to take the existing typography stuff and move that into components like @poofichu already did on `wox-ui`.